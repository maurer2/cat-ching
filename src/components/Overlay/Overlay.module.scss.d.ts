import globalClassNames from '../../style.d';
declare const classNames: typeof globalClassNames & {
  readonly overlay: 'overlay';
  readonly wrapper: 'wrapper';
  readonly overlayText: 'overlayText';
  readonly resetButton: 'resetButton';
};
export = classNames;
