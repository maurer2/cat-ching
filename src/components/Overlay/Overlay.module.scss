.overlay {
  position: absolute;
  top: 0;
  bottom: 0;
  left: 0;
  right: 0;
  display: flex;
  justify-content: center;
  align-items: center;
  backdrop-filter: blur(3px) grayscale(100%);
}

.overlayText {
  margin: 0;
  padding: 1rem;
  font-size: 2rem;
  background: white;
}
