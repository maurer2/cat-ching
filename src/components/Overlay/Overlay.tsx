import React from 'react';

import style from './Overlay.module.scss';
// import * as Types from './Overlay.types';

function Overlay(): JSX.Element {
  return (
    <aside className={style.overlay}>
      <p className={style.overlayText}>
        Purrfect!
      </p>
    </aside>
  );
}

export default Overlay;
