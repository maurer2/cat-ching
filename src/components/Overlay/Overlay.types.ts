export type OverlayProps = Record<string, never>;
