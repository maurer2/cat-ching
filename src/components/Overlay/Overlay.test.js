import React from 'react';
import { shallow } from 'enzyme';

import Overlay from './Overlay';

describe('Overlay', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Overlay/>);

    expect(wrapper).toMatchSnapshot();
  });
});
