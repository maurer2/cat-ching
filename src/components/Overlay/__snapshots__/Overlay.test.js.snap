// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overlay should render correctly 1`] = `
<aside
  className="overlay"
>
  <p
    className="overlayText"
  >
    Purrfect!
  </p>
</aside>
`;
